validation-test/Sema/rdar143582881.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
+// RUN: %target-swift-frontend -typecheck %t/Main.swift %t/Other.swift -enable-upcoming-feature MemberImportVisibility
```

Comment:
I forgot this needs a `REQUIRES: swift_feature_MemberImportVisibility`

Reply:
It might also be good to have a targeted test that doesn't depend on the SDK?

Reply:
Thank you for helping with that!